docs/interactions/structured-messages/carousel.md

```diff
@@ -118,8 +118,10 @@ Primary parameters are used by default, however, some parameters are unique or o
 
 | API Property | Specificity |
 |-|-|
+| **`structured_content.target`** | **Optional**. **Ignored** when the `url` is empty. Behavior applied when clicking on the url. Can be `webview` to open url on a [webview](../web-messaging/webview) above the chat, `open` to open url in new tab, or  `current` to open url in current tab. Defaults to **open** when not specified.|
```

Comment:
@aymebou I don't think this one applies to carousel

Reply:
Oh yeah sorry I got it wrong when solving the conflict